clouddriver-cloudrun/src/main/java/com/netflix/spinnaker/clouddriver/cloudrun/model/CloudrunMetaData.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2023 OpsMx, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.cloudrun.model;
+
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class CloudrunMetaData {
+
+  private String name;
+
+  private String namespace;
+
+  private String selfLink;
+
+  private String uid;
+
+  private String resourceVersion;
+
+  private long generation;
+
+  private String creationTimestamp;
+
+  private CloudrunMetadataLabels labels;
+
+  private CloudrunMetadataAnnotations annotations;
+
+  public String getName() {
```

Comment:
Can we use Lombok `@Getter & @Setter` here?

Reply:
@mattgogerly Please can you review the changes?

Reply:
@mattgogerly Please can you review the changes?